### DISCUSSION TITLE
How feasible would it be to add more button/input sizes via SCSS?

### DISCUSSION THREAD

**Discussion by PierBover** [twbs/bootstrap#34847] [Q&A]
I briefly looked into this and it seems I'd need to hand create new variables for each size, but what else would I need to do?

Is there any tutorial available?

**Comment by ZuBB** [comment]
Hey @PierBover did you manage to solve your issue?

(sorry for bumping an old thread)

**Reply by PierBover** [reply]
Hey

No, not really. Ended up moving to Bulma.

**Comment by ZuBB** [comment]
> Hey
> 
> No, not really. Ended up moving to Bulma.

Oh, good for you. Anyway, thanks for so quick response
